skpro/distributions/trafo/_transformed.py

```diff
@@ -62,14 +64,16 @@ class TransformedDistribution(BaseDistribution):
     def __init__(
         self,
         distribution,
-        transform,
+        transformer,
```

Comment:
sorry, we cannot change the names of already released arguments, or it might break user code. Not without deprecation and warnings ate least.